### ISSUE TITLE
[PHP] improve SDK env var docs

### ISSUE THREAD

**Issue by brettmc** [open-telemetry/opentelemetry.io#3959]
Could you add a bit more detail in the descriptions as to what users can accomplish by fiddling with these? In a separate PR, if you prefer.

_Originally posted by @theletterf in https://github.com/open-telemetry/opentelemetry.io/pull/3952#discussion_r1481173745_

**Comment by theletterf** [comment]
@brettmc Did the previous PR solve the issue?

**Comment by brettmc** [comment]
> Did the previous PR solve the issue?

No, I think more detail was expected in the original request. I'll take a shot at improving that section.
